samples/detect.v1p2beta1.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function detectPdfText(bucketName, fileName) {
```

Comment:
When I scan the sample, I don't see a clear reference to any PDF files. It's unclear to me what `gcsSourceUri` is and where the value comes from.

How come the function doesn't take a local PDF file as an argument?

Reply:
TODO: Add parameter comments for function parameters

For [example](https://github.com/googleapis/nodejs-vision/blob/master/samples/detect.js#L59):

```js
/**
 * TODO(developer): Uncomment the following lines before running the sample.
 */
// const bucketName = 'Bucket where the file resides, e.g. my-bucket';
// const fileName = 'Path to file within bucket, e.g. path/to/image.png';
```

Reply:
Fair point. I tried to match the function signature to the other detectXYZ functions in this repo which mostly take a bucketName and a fileName. This can be changed if it is confusing.

I tried to run it with a local file, but it did not work. It seems the only way to provide the input is through GCS at this point.

I also looked at the example. I am not quite clear what the comment means (if the const lines are uncommented, won't they overwrite the input parameters)? But I am fine to add them to the code. Maybe you can elaborate what the parameter comments provide?

Reply:
> I also looked at the example. I am not quite clear what the comment means (if the const lines are uncommented, won't they overwrite the input parameters)

Yep! That's the point :)

Developers copy/paste this code _(which doesn't include the function, only the contents of the region block)_ and they uncomment those lines to set the variable values

Reply:
> I tried to run it with a local file, but it did not work. It seems the only way to provide the input is through GCS at this point.

Makes some sense. Meh.

_Because_ it's not local (whereas most other Vision samples provide a local file version), I wonder if we should be a bit more explicit with the variable names (I did not find it obvious)

```js
/**
 * TODO(developer): Uncomment the following lines before running the sample.
 */
// const bucketName = 'Bucket where the file resides, e.g. my-bucket';
// const pdfFileName = 'Path to PDF file within bucket, e.g. path/to/document.pdf';
```

Just a thought

Reply:
Got it. Good point. My start tag is after the function signature. I will add the comments now.